vagga.yaml

```diff
@@ -54,8 +54,13 @@ containers:
       # Git is needed for pre-commit in runtime. Ref:
       # github.com/pre-commit/pre-commit/issues/456#issuecomment-269653630
       - !Install [git]
-      - !EnsureDir /pre-commit
-      - !Sh HOME=/pre-commit pre-commit install --install-hooks
+      - !Sh |
+            HOME=/root pre-commit install --install-hooks
+            rm -f /work/.git/hooks/pre-commit
```

Comment:
It won't as pre-commit creates a backup of your hook in `.git/hooks/pre-commit.legacy` while placing its own scripts in `.git/hooks/pre-commit`